models/hashflow/hashflow_crosschain_trades.sql

```diff
@@ -0,0 +1,26 @@
+{{
+    config(
+        alias="cross_chain_trades"
+        ,post_hook='{{ expose_spells(\'["ethereum", "avalanche_c", "bnb"]\',
+                        "project",
+                        "hashflow",
+                        \'["BroderickBonelli"]\') }}'
+    )
+}}
+
+{% set cross_chain_models = 
+    [
+        ref('hashflow_avalanche_c_crosschain_trades')
+        ,ref('hashflow_ethereum_crosschain_trades')
+        ,ref('hashflow_bnb_crosschain_trades')
+    ]
+%}
+
+{% for ref in cross_chain_models %}
+SELECT 
+    *
```

Comment:
Hi @BroderickBonelli,
could you please write full column names in this roll-up spell instead of `*`?

It can avoid mismatch of columns.

Reply:
Hi @Hosuke, no problem, just amended to list out column names.